stock_quant_package_number/models/stock_quant_package.py

```diff
@@ -0,0 +1,23 @@
+# Copyright 2025 Camptocamp SA
+# @author: Italo LOPES <italo.lopes@camptocamp.com>
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl)
+
+from odoo import api, fields, models
+
+
+class StockQuantPackage(models.Model):
+    _inherit = "stock.quant.package"
+
+    package_number = fields.Char(compute="_compute_package_number", store=True)
+    order_number = fields.Char(size=32)
+    sheet_number = fields.Integer()
+
+    @api.depends("order_number", "sheet_number")
+    def _compute_package_number(self):
+        for package in self:
+            if package.order_number and package.sheet_number:
+                package.package_number = (
+                    f"{package.order_number}_{package.sheet_number}"
+                )
+            else:
+                package.package_number = self.env._("Unknown Pack Number")
```

Comment:
Is it a good idea here to have a string? And with translation?
How should we then search or group by this stored field? There will be lots of different strings meaning the same.
Why not just empty?